docs/cne-install.md

```diff
@@ -0,0 +1,189 @@
+# SAT Upgrade with CNE Installer
+
+## Upgrade the System Admin Toolkit Product Stream
+
+Describes how to upgrade the System Admin Toolkit (SAT) product
+stream by using the Compute Node Environment (CNE) installer (`cne-install`).
+
+The CNE installer can only upgrade the SAT product stream for this release. See
+[Install the System Admin Toolkit Product Stream](install.md) for installation
+instructions.
+
+Upgrading SAT with `cne-install` is recommended because the process is both
+automated and logged to help you save time. The CNE installer can be used to
+upgrade SAT alone or with other supported products. Refer to the [*HPE Cray EX
+System Software Getting Started Guide (S-8000)*](https://www.hpe.com/support/ex-S-8000)
+for detailed information about `cne-install` and its options.
+
+### Prerequisites
+
+- CSM is installed and verified.
+- There must be at least 2 gigabytes of free space on the manager NCN on which
+the procedure is run.
+
+### Notes on the Procedures
+
+- Ellipses (`...`) in shell output indicate omitted lines.
+- In the examples below, replace `x.y.z` with the version of the SAT product stream
+being upgraded.
+- 'manager' and 'master' are used interchangeably in the steps below.
+
+### Pre-Upgrade Procedure
+
+1. Start a typescript and set the shell prompt.
+
+   The typescript will record the commands and the output from this upgrade.
+   The prompt is set to include the date and time.
+
+   ```screen
+   ncn-m001# script -af product-sat.$(date +%Y-%m-%d).txt
+   ncn-m001# export PS1='\u@\H \D{%Y-%m-%d} \t \w # '
+   ```
+
+### Upgrade Procedure
+
+1. Copy the release distribution gzipped tar file to `ncn-m001`.
+
+   Whether you are upgrading SAT alone or with other supported products, copy
+   the file in the same media directory as all other supported products.
+
+   **NOTE:** The `cne-install` command installs all files in the media directory
+   by default. If you are upgrading SAT alone, ensure only the SAT tarball is in
+   the media directory.
+
+1. Run the CNE installer.
```

Comment:
I think this should probably be split into two different use cases.  a) if you are installing SAT along with other software in which case i think we could simply say that SAT will be installed on the management nodes at the successful completion of the ncn_personalization phase.  and b) if you are just installing SAT ala-cart, then you want to run all the steps SKIPPING the sat_bootprep_cn stage since you (likely) don't wish to create compute images

Reply:
Wendy Palm left suggestions in [STP-3310](https://jira-pro.its.hpecorp.net:8443/browse/STP-3310) after speaking with John that I will be adding.